_rules/SC1-3-1-aria-required-owned-element.md

```diff
@@ -0,0 +1,243 @@
+---
+name: ARIA required owned elements
+
+description: |
+	The rule checks that a role has at least one of its required owned elements.
+
+success_criterion:
+- 1.3.1 # Info and Relationships (A)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Audrey Maniez
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any HTML or SVG element that is [exposed to assistive technologies](#exposed-to-assistive-technologies) and has an explicit [semantic role](#semantic-role) that has [required owned elements](https://www.w3.org/TR/wai-aria-1.1/#mustContain) listed for that role in [WAI-ARIA](https://www.w3.org/TR/wai-aria).
+
+### Expectation
+
+For each test target at least one instance of one [required owned element](https://www.w3.org/TR/wai-aria-1.1/#mustContain) listed for that role in [WAI-ARIA](https://www.w3.org/TR/wai-aria) is present as a [semantic role](#semantic-role) for an element [owned](https://www.w3.org/TR/wai-aria-1.1/#dfn-owned-element) by the test target.
+
+**Note:**
+- When a widget is missing required owned elements due to script execution or loading, authors [MUST]((w3.org/TR/wai-aria-1.1/#mustContain)) mark a containing element with `aria-busy` equal to `true`.
```

Comment:
Is this a note, part of the applicability, or an assumption? I'm never sure where to put what. However, this seems to be quite a substantive limitation so that a note may be too little. If there is a normative MUST then this may even need to be (part of) another test to check appropriate use of WAI-ARIA?

Reply:
I think we actually discussed if this should be a separate (semiautomated) rule under a composite rule along with this...

Reply:
@nitedog - In this case the note is a part of the expectation, to me it sits right with expectation, as we are saying, it is expected to be decorated with `aria-busy` under some conditions....

@annethyme - We did discuss this, let us discuss again in the rule writing session, to see if a new semi-automated rule is necessary for this. 

Otherwise, do not see the above comment as a blocker to progress with this rule.